### DISCUSSION TITLE
Using $HOSTNAME on the config.yaml to identify the server

### DISCUSSION THREAD

**Discussion by edsongper71** [open-telemetry/opentelemetry-collector-contrib#5198] [Q&A]
I'm working to configure OTEL Collector for Splunk and Dynatrace. I'm trying to add the configuration on Config.YAML to retrieve the server hostname using an ENV Variable like $HOSTNAME.

resource: attributes:

        key: host.name 
        action: insert 
        value: ${HOSTNAME}

    No errors to start the Agent.
    On Splunk side, the host is showing a blank-->  host:

    My idea is to have a generic configuration file to be distributed during the server provisioning.

   Edson

**Comment by skamarajugadda-clgx** [comment]
@edsongper71 I'm on the same page as you are let me know if anything worked for you
